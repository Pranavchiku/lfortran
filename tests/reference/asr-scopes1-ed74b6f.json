{
    "basename": "asr-scopes1-ed74b6f",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/scopes1.f90",
    "infile_hash": "48ffe7d9bb1021077b7bec62634ec2682cbfc211123626d338ba0c9e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-scopes1-ed74b6f.stdout",
    "stdout_hash": "c46afe6006adb86857b4243132c9081cbe9ae55ba726016d74e692f0",
    "stderr": "asr-scopes1-ed74b6f.stderr",
    "stderr_hash": "e6836719a3981c195d0ff5b89cd35fe78c01ba8abbaaafeb640eb74c",
    "returncode": 0
}