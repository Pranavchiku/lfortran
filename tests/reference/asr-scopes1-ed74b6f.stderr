warning: Implied save feature
 --> tests/scopes1.f90:3:1
  |
3 | integer :: i, j
  | ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/scopes1.f90:3:1
  |
3 | integer :: i, j
  | ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/scopes1.f90:10:5
   |
10 |     integer, intent(out) :: b
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
